versioned_docs/version-2.0.0/quickstart/java-spring-postgres.md

```diff
@@ -130,17 +132,25 @@ Here we just need to change the command used to start the application.
 keploy record -c "docker compose up" --container-name javaApp --build-delay 100
 ```
 
+<img src="/docs/img/pet-clinic-record.png" alt="Sample Keploy Record Java" width="100%" style={{ borderRadius: '5px' }} />
+
+
 ## Running the testcases using Keploy
 
 ```bash
 keploy test -c "docker compose up" --container-name javaApp --build-delay 50 --delay 20
 ```
 
+<img src="/docs/img/pet-clinic-test1.png" alt="Sample Keploy Test Java" width="100%" style={{ borderRadius: '5px' }} />
+
+
```

Comment:
Two images are together. Please add a one line on how both are different.

Reply:
hey any update on this? This was fixed